templates/CSI/cephfs/pod.yaml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Thinking maybe we should have pod.yaml generic for CepFS and RBD and resides under CSI dir.